api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestPipeline.java

```diff
@@ -1390,4 +1390,72 @@ public void onNewDataReceived(TensorsData data) {
             fail();
         }
     }
+
+    @Test
+    public void testSNPE() {
+        if (!NNStreamer.isAvailable(NNStreamer.NNFWType.SNPE)) {
+            /* cannot run the test */
+            return;
+        }
+
+        File[] model = APITestCommon.getSNPEModel();
+        String desc = "appsrc name=srcx ! " +
+                "other/tensor,dimension=(string)3:299:299:1,type=(string)float32,framerate=(fraction)0/1 ! " +
+                "tensor_filter framework=snpe " + "model=" + model[0].getAbsolutePath() + " ! " +
+                "tensor_sink name=sinkx";
+
+        try (
+            Pipeline pipe = new Pipeline(desc);
+            TensorsInfo info = new TensorsInfo()
+        ) {
```

Comment:
This indent makes some confusion.

Reply:
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
This Java doc describes this kind of statement. I think the current indent is compatible with conventions.

Reply:
Yes, I though this was connected to line 1436. My bad.